ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -1,8 +1,69 @@
 module DynamicQuantitiesLinearAlgebraExt
 
-import LinearAlgebra: norm
-import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity
+import LinearAlgebra: norm, inv, (\), svd, Algorithm, default_svd_alg, SVD, Diagonal, Adjoint, Transpose, AbstractRotation, AbstractMatrix, eigen, eigsortby, Eigen, det
+import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity, AbstractDimensions, QuantityArray, Quantity
```

Comment:
I learned this the other day and haven't yet put it in for the rest of the code, but this should be 
```suggestion
import LinearAlgebra: norm, svd, Diagonal, eigen, det
using LinearAlgebra: Algorithm, default_svd_alg, SVD, Adjoint, Transpose, AbstractRotation, AbstractMatrix, eigsortby, Eigen
using DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity, AbstractDimensions, QuantityArray, Quantity
```
because `import` should technically only be used if you are overloading a method.

Meanwhile the `inv` and `\` definitions should go in the main codebase since these are exported by `Base` rather than `LinearAlgebra` (they are only overloaded there, rather than given their original definitions).

Also, `AbstractMatrix` is already available in Julia, so does not need the import.

Also, some of these imports are not used – `Eigen`